### PR TITLE
feat(onyx-1276): restore progressive onboarding tooltips for new home view

### PR DESCRIPTION
**Pull request by nickskalkin**
This PR resolves [ONYX-1276]

### Description

Restores missing tooltips:

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#nochangelog

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[ONYX-1276]: https://artsyproduct.atlassian.net/browse/ONYX-1276?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ✅ **No changelog changes**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against dc8f24c5a489c58d62125fff720711c31cafb6f9
</p>

File: src/app/Scenes/HomeView/HomeView.tsx [hunk 1]
```diff
@@ -8,30 +8,44 @@ import {
...
 
   const { data, loadNext, hasNext } = usePaginationFragment<
     HomeViewQuery,
     HomeViewSectionsConnection_viewer$key
   >(sectionsFragment, queryData.viewer)
 
-  const flatlistRef = useBottomTabsScrollToTop("home")
+  const meData = useFragment<HomeView_me$key>(meFragment, queryData.me)
+  const savedArtworksCount = meData?.counts?.savedArtworks ?? 0
+  useDismissSavedArtwork(savedArtworksCount > 0)
```
Comment (anandaroop) [inline src/app/Scenes/HomeView/HomeView.tsx:47]: **question (non-blocking):** This simply suppresses a saves-related tooltip, if the user has already saved, yes?

**Review by anandaroop — APPROVED** [review]
👍🏽

File: src/app/Scenes/HomeView/HomeView.tsx [hunk 2]
```diff
@@ -170,6 +192,10 @@ const sectionsFragment = graphql`
 
 export const homeViewScreenQuery = graphql`
   query HomeViewQuery($count: Int!, $cursor: String) {
+    me {
+      ...HomeView_me
+    }
```
[the remark this answers is not in the document]
Comment (nickskalkin) [reply src/app/Scenes/HomeView/HomeView.tsx:197]: @MounirDhahri this data (saves count) is used by `useDismissSavedArtwork` hook. It is not recommended to use a hook inside another one right? What other approach would you recommend? Another option could be to have another query renderer which solely purpose is to query this data and execute this hook, but I think there should be a better approach.

I will merge this PR in the meantime, but let's continue this discussion

File: src/app/Scenes/HomeView/HomeView.tsx (hunk 1 above)
Comment (nickskalkin) [reply src/app/Scenes/HomeView/HomeView.tsx:47]: yes

**Pull request merged**